### PR TITLE
Split all resume items into react components

### PR DESCRIPTION
**Pull request by DTThor**
Next I will convert CSS styling to Stylus

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,77 +1,26 @@
...
-          <h5>EDUCATION</h5>
-          <h6>BA in Public Relations</h6>
-          <h6 className="school">Pepperdine University</h6>
-          <h6 className="school">2008-2012</h6>
-
-        </div>
-
+        <h3 className="container-item"><i className="icon fas fa-user-circle fa-lg"></i> CAREER PROFILE</h3>
+        <Profile />
+        <h3 className="container-item"><i className="icon fas fa-briefcase fa-lg"></i> EXPERIENCE</h3>
```
Comment (lucas-aragno) [inline src/App.js]: this `h3` it's repeating a log maybe make a component that takes the title (career profile), and the icon (brief case, circle)) as props?

File: src/components/ExperienceCard.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+
+const ExperienceCard = ({ role, company, details, id }) => {
+  return (
```
Comment (lucas-aragno) [inline src/components/ExperienceCard.js]: can you make this function auto return? (remove the return and change the {} with ())

File: src/components/ExperienceList.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ExperienceCard from './ExperienceCard';
+
+const ExperienceList = () => {
+  const experience = [
```
Comment (lucas-aragno) [inline src/components/ExperienceList.js]: can you move this experience to a json or js file ourside the component?

File: src/components/Profile.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+const Profile = () => {
+  return (
```
Comment (lucas-aragno) [inline src/components/Profile.js]: ditto w/ the return statement

File: src/components/Sidebar.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+
+const Sidebar = () => {
```
Comment (lucas-aragno) [inline src/components/Sidebar.js]: ditto

**Review by lucas-aragno — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (DTThor) [reply src/App.js]: good idea, just made this change.

File: src/components/ExperienceCard.js (hunk 2 above)
Comment (DTThor) [reply src/components/ExperienceCard.js]: done

File: src/components/ExperienceList.js (hunk 3 above)
Comment (DTThor) [reply src/components/ExperienceList.js]: done

File: src/components/Profile.js (hunk 4 above)
Comment (DTThor) [reply src/components/Profile.js]: done

File: src/components/Sidebar.js (hunk 5 above)
Comment (DTThor) [reply src/components/Sidebar.js]: done

File: package.json [hunk 6]
```diff
@@ -40,6 +40,7 @@
     "eslint-plugin-node": "^6.0.1",
     "eslint-plugin-promise": "^3.8.0",
     "eslint-plugin-react": "^7.9.1",
-    "eslint-plugin-standard": "^3.1.0"
+    "eslint-plugin-standard": "^3.1.0",
+    "json-loader": "^0.5.7"
```
Comment (lucas-aragno) [inline package.json]: can you move this to the regular dependencies? you will need it to build

File: src/components/ExperienceList.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import ExperienceCard from './ExperienceCard'
+import experience from '../experience.js'
+
+const ExperienceList = () => {
+  return (
```
Comment (lucas-aragno) [inline src/components/ExperienceList.js]: can you change the return style in here and on the maps to match the other functions ? `(user, i) => (.....)`

File: src/components/SectionHeader.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+const SectionHeader = (props) => {
+  return (
```
Comment (lucas-aragno) [inline src/components/SectionHeader.js]: ditto

File: src/components/SectionHeader.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+    <div className="container container-item">
+      <h3 className="container-item"><i className={props.icon}></i> {props.name}</h3>
```
Comment (lucas-aragno) [inline src/components/SectionHeader.js]: can you do `({name, icon})` on the params?

**Review by lucas-aragno — CHANGES_REQUESTED** [review]

**Review by lucas-aragno — APPROVED** [review]

**Pull request merged**